### PR TITLE
AWS-3584: adding in support to allow us to automatically populate instance profiles

### PR DESCRIPTION
**Pull request by NatahlieBeavers**
#### Reviewers! 
Here's [How to do a Code Review](https://cloudhealthtech.atlassian.net/wiki/spaces/EN/pages/17989636/Code+Review)

#### Deploy readiness:

- [ ]  2 Reviews: 1 peer/buddy and 1 SME from [here](https://cloudhealthtech.atlassian.net/wiki/spaces/EN/pages/17989636/Code+Review)
- [ ]  Did you add the JIRA ticket ID to the PR title? 
- [ ]  Is your PR title **meaningful?** `[ENG-3333] Update pull request template`, not `Typo Fix` or `Add test`
- [ ]  After commits are squashed, will your primary commit message be meaningful?

#### Documentation links:

- JIRA Ticket(s):https://cloudhealthtech.atlassian.net/browse/AWS-3284
- Design Doc (where applicable):
- Link to passing Solano build (if PR tests are failing due to inter-repo dependencies):
  - [ ]  Branch pointers in Gemfile removed?

#### Proposed changes:
- Create new process, run by rake task, to automatically populate aws_instance_type_profiles if a new instance type is added
-*NOTE* This PR right now does *NOT* include code to allow updates of existing instance profiles if what we have in the table does not match the api. This will be a feature that will be added.

_Rollback Plan:_

-Revert

### CODE REVIEW

File: lib/amazon-pricing/definitions/ec2-instance-type.rb [hunk 1]
```diff
@@ -143,6 +143,33 @@ def self.get_network_mbps(throughput)
...
+        when 'High'
+          throughput = :high
+        when '10 Gigabit','Up to 10 Gigabit'
+          throughput = :ten_gigabit
+        when '20 Gigabit'
+          throughput = :twenty_gigabit
+        when '25 Gigabit'
+          throughput = :twentyfive_gigabit
+        else
+          binding.pry
```
Comment (stanmluke) [inline lib/amazon-pricing/definitions/ec2-instance-type.rb]: debug.
is the return expected to be a 'symbol' or a 'string'?

Comment (NatahlieBeavers) [reply lib/amazon-pricing/definitions/ec2-instance-type.rb]: We convert it to a string below. The network MBPS gives us a int value.

File: lib/amazon-pricing/definitions/ec2-instance-type.rb [hunk 2]
```diff
@@ -143,6 +143,33 @@ def self.get_network_mbps(throughput)
       network_mbps
     end
 
+    # Take in string from amazon pricing api, return network properties
```
Comment (NatahlieBeavers) [inline lib/amazon-pricing/definitions/ec2-instance-type.rb]: add in note about input and output types

File: lib/amazon-pricing/definitions/ec2-instance-type.rb [hunk 3]
```diff
@@ -143,6 +143,35 @@ def self.get_network_mbps(throughput)
...
+    # Input: String containing network capacity from amazon-pricing-api
+    # Output: network throughput as a string, int containing network mbps
+    def self.get_network_information(network_string)
+      throughput = nil
+      case network_string
```
Comment (jeffzhou791) [inline lib/amazon-pricing/definitions/ec2-instance-type.rb]: can we convert all the case when's to a hash, just like @Network_Throughput_MBits_Per_Second?

**Pull request merged**